### PR TITLE
Build pod lib by a Gradle Task

### PR DESCRIPTION
**Pull request by magnumrocha**
Some enhacements:
- build pod lib by gradle tasks
- moved podspec to native module

### CODE REVIEW

File: native/StarterCore.podspec [hunk 1]
```diff
@@ -4,14 +4,14 @@ Pod::Spec.new do |s|
...
+  s.license                 = { :type => 'MIT', :file => '../LICENSE' }
   s.platform                = :ios
   s.requires_arc            = true
   s.module_name             = "EX"
   s.source                  = { :git => 'https://github.com/noheltcj/multiplatform-kotlin-hackathon-starter.git', :tag => '0.0.1' }
-  s.source_files            = 'build/native/release/StarterCore.framework/Headers/*.h'
-  s.resource                = 'build/native/release/StarterCore.framework/Info.plist'
-  s.public_header_files     = 'build/native/release/StarterCore.framework/Headers/*.h'
+  s.source_files            = 'build/release/StarterCore.framework/Headers/*.h'
+# s.resource                = 'build/release/StarterCore.framework/Info.plist'
```
Comment (noheltcj) [inline native/StarterCore.podspec]: Good call. Might as well remove the line entirely.

File: native/build.gradle [hunk 2]
```diff
@@ -44,3 +44,40 @@ konanArtifacts {
 dependencies {
     expectedBy project(':common')
 }
+
+task buildPod(type: Sync, dependsOn: 'build') {
```
Comment (noheltcj) [inline native/build.gradle:48]: 👍

**Comment by noheltcj** [comment]
This looks great, I'll finish reviewing tonight.

File: native/build.gradle [hunk 3]
```diff
@@ -44,3 +44,40 @@ konanArtifacts {
...
+            workingDir '.'
+            commandLine 'sh', '-c', "lipo -create " +
+                    "-output build/release/StarterCore.framework/StarterCore " +
+                    "build/konan/bin/ios_arm64/StarterCore.framework/StarterCore " +
+                    "build/konan/bin/ios_x64/StarterCore.framework/StarterCore"
+        }
+
+        exec {
+            workingDir '.'
+            commandLine 'sh', '-c', 'pod lib  lint --verbose'
```
Comment (noheltcj) [inline native/build.gradle]: extra space here between lib and lint

**Review by noheltcj — APPROVED** [review]
Looks good! Feel free to squash merge with a comment similar to what you opened the PR with. I have some README changes prepared once you merge.

**Comment by magnumrocha** [comment]
Hi @noheltcj, all your comments were resolved, I think this PR is ready to be merged..., I have no rights to do that on your repo.

**Pull request merged**